numpy/core/src/umath/loops_logical.dispatch.c.src

```diff
@@ -0,0 +1,427 @@
+/*@targets
+ ** $maxopt baseline
+ ** neon asimd
+ ** sse2
+ **/
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+#include "lowlevel_strided_loops.h"
+// Provides the various *_LOOP macros
+#include "fast_loop_macros.h"
+
+/*******************************************************************************
+ ** Extra SIMD intrinsics
+ ******************************************************************************/
+
+#if NPY_SIMD
+#if !defined(NPY_HAVE_SSE2)
+#define USE_NPYV_REDUCE_MINMAX
+#endif
```

Comment:
```suggestion
```
no need for two paths, new intrinsics any/all covered by #22306 should provide fair optimization across all SIMD extensions